aiomqtt/queue.py

```diff
@@ -0,0 +1,88 @@
+"""
+trio/anyio no longer have queues, but sometimes a memory object stream is
+unwieldy. Thus this re-implemens a simple queue using
+`anyio.create_memory_object_stream`.
+"""
+
+from __future__ import annotations
+
+import anyio
+from anyio import create_memory_object_stream
+
+from outcome import Error, Value
```

Comment:
I didn't see `outcome` in the dependencies. I also don't think it's worth the trouble to implement a wrapper on top of memory object streams.